roottest/root/io/alloc/CMakeLists.txt

```
@@ -1,7 +1,25 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOTTEST_GENERATE_DICTIONARY(customConstructor
+                             customConstructor.C
+                             LINKDEF linkdef.h
+                             NO_ROOTMAP NO_CXXMODULE
+                             FIXTURES_SETUP root-io-alloc-customConstructor-fixture)
+
+if(ClingReinstateTemplateRootIOCtor)
+  set(testref customConstructor.ref)
+else()
+  set(testref customConstructor.ref-notemplateCtor)
+endif()
+
+ROOTTEST_ADD_TEST(customConstructorLib
+                  MACRO runcustomConstructor.C
+                  OUTREF ${testref}
```

```suggestion
                  OUTREF ${testref}
                  ${conversion}
```
See connected change on line 8.

---

Instead I add `OUTCNVCMD ${conversion}`